docs/android/gradle.md

```diff
@@ -53,22 +45,57 @@ In `build.gradle` add the PS maven artifactory to the repositories for project a
         dependencies { ... }
     }
 
-    allprojects {
-        repositories {
-            maven {
-                url = uri("https://artifactory.productscience.app/releases")
-            }
+    ```
+
+## 3. Add Product Science dependencies to buildscript in root **build.gradle**
+
+In the `buildscript` block of the root **build.gradle** file, add the classpaths for the productscience
+`transformer-plugin` and `transformer-instrumentation` artifacts:
+
+=== "Groovy"
+    ```groovy title="build.gradle"
+    buildscript {
+        repositories { ... }
+        dependencies {
+            classpath "com.productscience.transformer:transformer-plugin:{{ android_release() }}"
+            classpath "com.productscience.transformer:transformer-instrumentation:{{ android_release() }}"
         }
     }
+    ...
+    ```
 
+=== "Kotlin DSL"
+    ```kotlin title="build.gradle.kts"
+    buildscript {
+        repositories { ... }
+        dependencies {
+            classpath("com.productscience.transformer:transformer-plugin:{{ android_release() }}")
+            classpath("com.productscience.transformer:transformer-instrumentation:{{ android_release() }}")
+        }
+    }
+    ...
     ```
 
-If the project is configured to prefer settings repositories maven source should be added to settings file:
+
+## 4. Add Product Science maven repository url to all submodules
+
+The productscience plugin will attempt to instrument all modules of the app, 
+so it is necessary to provide access to dependencies for all submodules. 
+
+To achieve that, product science maven repository has to be visible for all submodules.
+This can be achieved in Gradle in two ways: either using the `allprojects` block in the root **build.gradle**, 
+or the `dependencyResolutionManagement` block in the **settings.gradle**.
+
+
+### Case 1: Project uses **settings.gradle** to specify module dependencies (requires gradle 6.8 or later)
```

Comment:
To match verbiage above:
Using the **settings.gradle** file to specify module dependencies ...